SU-SWS/stanford_profile
# D8CORE-2856 Add list and entity reference paragraph types

# READY FOR REVIEW

# Summary
- Created entity reference and view reference paragraph types
- Stubbed out starting point for a "Card" display for each paragraph type. These will be revised later
- Limited the views to only a few choices that I assume are the best at this time.

# Need Review By (Date)
- 10/19

# Urgency
- medium

# Steps to Test
1. checkout this branch and the same branch in https://github.com/SU-SWS/stanford_profile_helper/pull/34
1. `composer require drupal/viewfield` (it's easier this way than doing all the different requirements)
1. `blt drupal:update` or `blt drupal:install -n`
1. Create a basic page and use the "Lists" or "Content Reference" paragraph types.
1. verify they work close to what you'd expect.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

see https://github.com/SU-SWS/stanford_profile_helper/pull/34
